php/pantheon/checks/objectcache.php

```diff
@@ -29,11 +29,14 @@ public function run() {
 		} else {
 			$this->alerts[] = array("message"=> "object-cache.php exists", "code" => 0);
 		}
-
 		if ( ! defined( 'WP_REDIS_OBJECT_CACHE' ) || ! WP_REDIS_OBJECT_CACHE ) {
```

Comment:
This conditional needs to be flipped. It's true we no longer care if it's _not_ defined, but now we care if it _is_ defined. However, I don't know that we care what the _value_ of the definition is, just that the constant exists.

```suggestion
		if ( defined( 'WP_REDIS_OBJECT_CACHE' ) ) {
```